### PR TITLE
REFS: #27 and fixes #27

### PR DESCRIPTION
**Pull request by Fi-ona**

### CODE REVIEW

**Comment by DarshiL7** [comment]
Day 15 Assignment
submit button sending request to the server. code should be at the client side only. do necessary modification

File: Modules/JavaScript/Day16/FiyonaMistry/Assignment/info.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                return false;
+            }
+        }
+    }
+    else
+    {
+        alert("Invalid date format!");
+        return false;
+    }
+}
```
Comment (MilitPanchasara) [inline Modules/JavaScript/Day16/FiyonaMistry/Assignment/info.js:57]: Year validation is not proper 02-29-0000 is a valid date according to code.

File: Modules/JavaScript/Day17/FiyonaMistry/Assignment/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            if(id.test(id_regEx) == true){
+                alert("Valid Employee Id");
+            }
+            else{
+                alert("Invalid Employee Id");
+            }
+        }
+    </script>
+</body>
+</html>
```
Comment (MilitPanchasara) [inline Modules/JavaScript/Day17/FiyonaMistry/Assignment/index.html:91]: error occurs in ID validation , can't see any error messages and validation rules.

**Review by MilitPanchasara — COMMENTED** [review]
Practice exercise 9 is missing

File: Modules/JavaScript/Day18/FiyonaMistry/Practice/index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            });
+            makePromise.then(alert);
+            makePromise.catch(
+                reject(new Error(err))
+            );
+            return false;
+        }
+    </script>
+</body>
+</html>
```
Comment (MilitPanchasara) [inline Modules/JavaScript/Day18/FiyonaMistry/Practice/index.html:74]: can't see reversed string in console.

**Pull request merged**